web/client-api/src/main/java/io/deephaven/web/client/api/barrage/def/TableAttributesDefinition.java

```diff
@@ -41,6 +45,10 @@ public boolean isInputTable() {
         return remainingAttributeKeys.contains(INPUT_TABLE_ATTRIBUTE);
     }
 
+    public boolean isStreamTable() {
+        return map.getOrDefault(STREAM_TABLE_ATTRIBUTE, "").equals("true");
```

Comment:
consider `"true".equals(map.get(STREAM_TABLE_ATTRIBUTE)` so you don't have to care about nulls